src/test/java/com/uid2/admin/vertx/test/ServiceTestBase.java

```diff
@@ -260,10 +264,8 @@ protected void setAdminUsers(AdminUser... adminUsers) {
         }
     }
 
-    protected static Handler<AsyncResult<HttpResponse<Buffer>>> expectHttpError(VertxTestContext testContext, int errorCode) {
-        return ar -> {
-            assertTrue(ar.succeeded());
-            HttpResponse response = ar.result();
+    protected static TestHandler<HttpResponse<Buffer>> expectHttpError(VertxTestContext testContext, int errorCode) {
```

Comment:
Not asking you to change this in this PR, but this method could really be called `expectHttpStatus` or similar, there's nothing specific to errors.

Reply:
I think this make more sense to call it `expectHttpStatus`, I've updated it